packages/walletconnect/src/wc-wallet.ts

```diff
@@ -48,13 +47,24 @@ const defaultInitOptions: SignClientTypes.Options = {
   },
 };
 
+const requiredNamespaces = {
+  eip155: {
+    chains: [
+      'eip155:44787', // alajores
+      'eip155:42220', // celo
+      'eip155:62320', // baklava
+    ],
+    methods: Object.values(SupportedMethods),
+    events: ['chainChanged', 'accountsChanged'],
+  },
+};
```

Comment:
is this saying what chains a dapp supports? If so we should probably build this from the list of networks that developers pass into the Celo provider

Reply:
Good point, yeah that's exactly what it is